### ISSUE TITLE
Error in non public Postgres schemas: The table does not exist in the current database.

### ISSUE THREAD

**Issue by zomars** [prisma/studio#871]
**Describe the bug**

Having a connection url like this:

```
postgresql://user:pass@database-XXX.amazonaws.com:5432/anotherName
```

Notice how we're not using the default `/public` which I think is causing this error.

**To Reproduce**
1. Go to https://cloud.prisma.io
2. Add a environment url that doesn't use public as database name
3. Try to run a query or use prisma studio

**Expected behavior**

Being able to run queries normally.

**Screenshots**

![image](https://user-images.githubusercontent.com/3504472/156647326-ff8c0846-ad47-421c-b429-de5ef63850b5.png)

**Desktop (please complete the following information):**
 - OS: macOS Monterrey
 - Browser Firefox

**Comment by janpio** [comment]
The `anotherName` is the database name, and the `public.` you see in the error message is the schema - which you could customize via `?schema=public` in the connection string. So everythign is in order here.

But it seems that the table `users` in schema `public` in the database `anotherName` does not exist. Ist that possible?

**Comment by zomars** [comment]
I've appended the `?schema=anotherName` to my DB url and now I'm getting this: 

![image](https://user-images.githubusercontent.com/3504472/159522873-0fa0b31d-be4a-439c-a625-c2ae00501f2b.png)

EDIT: Nevermind, adding `?schema=public` to my database URL solved it. 🙏🏽

**Comment by zomars** [comment]
Let's re-open this @janpio. The fatal issue is still appearing when opening Studio, (not on queries tho)

**Comment by janpio** [comment]
I am unclear what is going wrong then. If this is now about a new error message for `count`, then please open a new issue and provide all the relevant information and close this one again.

**Comment by zomars** [comment]
Closed in favor of #890
